less/hotgraphic.less

```diff
@@ -24,6 +24,23 @@
 				color: @item-color-hover;
 			}
 		}
+		&.number {
+			width: 48px;
+			height: 48px;
+			background-color: @item-color;
+			&.visited {
+				background-color: @item-color-visited;
+			}
+			.no-touch &:hover {
+				background-color: @item-color-hover;
+			}
+
+			.hotgraphic-graphic-number-icon {
+				font-size: 20px;
```

Comment:
Now that you've increased the size of the pin, we could bump up the `font-size` to something like `24px`.